libs/platform/src/lib/table/table.component.ts

```diff
@@ -212,6 +212,10 @@ export class TableComponent<T = any> extends Table<T> implements AfterViewInit,
     @Input()
     enableDragResize = true;
 
+    /** Whether to fix the table header and footer. Will be automatically set to true if therer are freezable columns.*/
```

Comment:
typo for `therer` and shouldn't it be "Will be automatically set to `false` if there are freezable columns.", since `fixed = false;` ?